cmd/posix_test.go

```diff
@@ -1021,20 +1022,11 @@ func TestPosixReadFile(t *testing.T) {
 			volume, "myobject",
 			-1, 5,
 			nil,
-			func() error {
-				if runtime.GOOS == globalWindowsOSName {
-					return &os.PathError{
-						Op:   "read",
-						Path: slashpath.Join(path, "success-vol", "myobject"),
-						Err:  syscall.Errno(0x57), // invalid parameter
-					}
-				}
-				return &os.PathError{
-					Op:   "read",
-					Path: slashpath.Join(path, "success-vol", "myobject"),
-					Err:  os.ErrInvalid,
-				}
-			}(),
+			&os.PathError{
```

Comment:
Can you explain a little why this change is needed ? it seems (from travis error log) that windows still returns "invalid parameter", is Linux behavior changed in 1.9 and returning "negative offset" instead ?

Reply:
Appveyor hasn't moved to go1.9 yet (even I specified in the configuration), hence the failure